gtsfm/evaluation/metrics_report.py

```diff
@@ -179,25 +250,38 @@ def get_html_header() -> str:
               </head>"""
 
 
-def generate_metrics_report_html(metrics_groups: List[GtsfmMetricsGroup], html_path: str) -> None:
+def generate_metrics_report_html(
+    metrics_groups: List[GtsfmMetricsGroup],
+    html_path: str,
+    colmap_files_dirpath: str,
+    metric_paths: List[str],
+) -> None:
     """Generates a report for metrics groups with plots and tables and saves it to HTML.
 
     Args:
         metrics_groups: List of metrics to be reported.
+        html_path: Path where this report is written to (default: output_dir/gtsfm_metrics_report.html).
+        colmap_files_dirpath: Optional; If a path to a directory containing a COLMAP reconstruction
+          (as cameras.txt, images.txt, and points3D.txt) is provided, the COLMAP metrics will also be
+          included in the report.
+        metric_paths: A list of paths to GTSfM metrics.
```

Comment:
are these Gtsfm metrics? or colmap?

Reply:
GTSfM Metrics, which will typically be GTSFM_MODULE_METRICS_FNAMES.